CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2022-01-17
```

Comment:
```suggestion
## [1.1] - (unreleased)
```

i want to do a minor version bump so that we have the possibility of doing security fixes as 1.0.x for php 7.2 in case something comes up.